### PR TITLE
add uv_poll_cb callback note regarding status == UV_EBADF

### PR DESCRIPTION
**Pull request by jensbjorgensen**

### CODE REVIEW

File: docs/src/poll.rst [hunk 1]
```diff
@@ -101,7 +101,8 @@ API
     with one of the `UV_E*` error codes (see :ref:`errors`). The user should
     not close the socket while the handle is active. If the user does that
     anyway, the callback *may* be called reporting an error status, but this is
-    **not** guaranteed.
+    **not** guaranteed. If `status == UV_EBADF` polling is discontinued for the
+    file handle, as if :c:func:`uv_poll_stop` had been called.
```
Comment (bnoordhuis) [inline docs/src/poll.rst:105]: "as if uv_poll_stop had been called" isn't strictly speaking true. Neither the Windows nor the UNIX implementation work exactly like that. It's close but there are small observable differences.

I'd replace it with something like "and no further events will be reported." Perhaps you also want to mention one should normally call `uv_close()` afterwards.

**Comment by jensbjorgensen** [comment]
done

File: docs/src/poll.rst [hunk 2]
```diff
@@ -101,7 +101,9 @@ API
     with one of the `UV_E*` error codes (see :ref:`errors`). The user should
     not close the socket while the handle is active. If the user does that
     anyway, the callback *may* be called reporting an error status, but this is
-    **not** guaranteed.
+    **not** guaranteed. If `status == UV_EBADF` polling is discontinued for the
+    file handle and no further events will be reported, the user should
```
Comment (bnoordhuis) [inline docs/src/poll.rst:105]:
```suggestion
    file handle and no further events will be reported. The user should
```

**Review by bnoordhuis — APPROVED** [review]
LGTM, thanks!

**Pull request merged**